metamapper/irs/coreir/ir.py

```diff
@@ -2,10 +2,1031 @@
 from hwtypes import BitVector, Bit
 from hwtypes.adt import Product
 from peak import Peak, name_outputs, family_closure, Const
-from peak.family import AbstractFamily
+from peak.family import AbstractFamily, MagmaFamily, SMTFamily
+from ...node import Nodes, Constant, DagNode, Select
+from hwtypes import SMTFPVector, FPVector, RoundingMode
+import magma
 
 def gen_peak_CoreIR(width):
```

Comment:
Can you split this file? Specifically group all the adhoc micro ops for floating point in a separate included file.